_topic_maps/_topic_map.yml

```diff
@@ -587,6 +587,8 @@ Topics:
   Topics:
   - Name: About disconnected environment updates
     File: index
+  - Name: Mirroring the OCP image repository
```

Comment:
We're not allowed to use the abbreviation "OCP" in our docs. Normally we'd use {product-title}, but that doesn't work for the topic map files. If you really want it here, you'll have to write out "OpenShift Container Platform"

Reply:
Sounds good, I'll write out "OpenShift Container Platform" instead